### DISCUSSION TITLE
My arduino doesn't connect to my laptop

### DISCUSSION THREAD

**Discussion by sw08** [NicoHood/HoodLoader2#90] [Q&A]
1. I burned this on my arduino. - It worked
2. I decided to burn the original bootloader on the arduino and did it
3. I think it successfully burned, but I can't find arduino port on device manager, arduino IDE, etc......
4. But I can upload with my other arduino. - (I used `Upload with programmer`)
5. And when I connect it to my laptop, I can't hear the sound (which we can hear when we connect USBs).
6. How can I solve it?

**Comment by NicoHood** [comment]
I dont understand you. Did you burn hoodloader2 or the original bootloader? And how did you burn/installed it?

This may help:
https://github.com/NicoHood/HoodLoader2/wiki/How-to-use-reset
